src/providers/config/config.spec.ts

```diff
@@ -46,28 +44,25 @@ describe('Provider: Config Provider', () => {
         });
       });
     });
-    it('should set default config if storage is empty', () => {
-      let defaults = configProvider.getDefaults();
-      persistenceProvider.clearConfig().then(() => {
-        configProvider.load().then(() => {
-          persistenceProvider.getConfig().then((config) => {
-            expect(config).toBeNull();
-            expect(defaults).not.toBeNull();
-          });
-        });
-      });
+    it('should set default config if storage is empty', async () => {
+      const defaults = configProvider.getDefaults();
+      await persistenceProvider.clearConfig();
+      await configProvider.load();
+      const config = await persistenceProvider.getConfig();
+      expect(config).toBeNull();
+      expect(defaults).not.toBeNull();
     });
     it('should set config from storage', () => {
-      persistenceProvider.getConfig().then((config) => {
+      persistenceProvider.getConfig().then(config => {
         expect(this.configCache).not.toBeNull();
       });
     });
-    it('should set default config if file is corrupted', async () => {
+    it('should set default config if file is corrupted', () => {
```

Comment:
It seems like the description for this test is no longer accurate, since the code being tested no longer seems to reset to the default config on a load error.